modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add Network Interface*.
+. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. Click *Add Network Interface* to add the network interface without restarting the virtual
+machine. Note that for the virtual machine to be accessible, it must be restarted.
+. Use the drop-down lists and check boxes to edit the network
+interface configuration.
+. Click *Save Changes*.
+. Click *OK*.
+
+The new network interface is listed at the top of the list of network interfaces
+until the user restarts it. The new network interface has a ’Pending VM restart’
+*Link State* until you restart the virtual machine. Hovering over the
+*Link State* will display more detailed information.
+
+The *Link State* is set to *Up* by default when the network interface card
+is defined on the virtual machine and connected to the network.
+
+// Scrubbing all conditionals used in module
+
+:object!:
```

Comment:
As with #18934 - This one will also need to be updated with:
ifeval::["{context}" == "cnv-edit-vms"]
:object!:
:object-gui!:
endif::[]

ifeval::["{context}" == "cnv-editing-vm-template"]
:object!:
:object-gui!:
endif::[]

Reply:
Added.